WcaOnRails/app/models/competition.rb

```diff
@@ -384,6 +387,22 @@ def warnings_for(user)
     warnings
   end
 
+  def reg_warnings(user)
```

Comment:
It's great you extracted that to a separate method, although it's not used anywhere now ^^ Also the method doesn't use the `user` argument, so you can remove it. Then it could be used [here](https://github.com/thewca/worldcubeassociation.org/blob/18d3bde220cdf8c804c37c61af2d92975fd40ae5/WcaOnRails/app/models/competition.rb#L386) like this:


```rb
if reg_warnings.any?
  warnings = reg_warnings.merge(warnings)
end
```

Reply:
I don't really see where it is still being called, but I tried implementing this. It seems this line is now what is causing a few issues for travis.